docs/app/docs-infra/hooks/use-code/page.mdx

```diff
@@ -98,7 +98,7 @@ interface UseCodeOpts {
   stackBlitzPrefix?: string; // StackBlitz URL prefix
   initialVariant?: string; // Initially selected variant
   initialTransform?: string; // Initially selected transform
-  fileHashMode?: 'full' | 'read' | 'remove'; // Controls URL hash management
+  fileHashMode?: 'full' | 'clean' | 'remove' | 'remove-after-interaction'; // Controls URL hash management
```

Comment:
I don't think need this needs to be an option, we should do it one way and one way only across all our documentation.

```suggestion
```